.gitignore

```diff
@@ -77,3 +77,4 @@ sftp-config.json
 target/
 var/
 venv/
+/locale/
```

Comment:
As a side note not strictly related to the contents of this PR:

Instead of only ignoring the repository root level locale directory, we should not generate translations for the third party packages in the first place. Adding `--ignore=venv/*` to `makemessages` command that should take care of that but it's easily forgotten if it needs to be manually added every time updating translations. Should there be a basic bash script for generating the locales in future?

Reply:
Good point. We could make a bash script or would it be enough to just have that command in the README to avoid extra custom scripts?

Reply:
Done. Added bash-script for translations.